tests/test_features.py

```diff
@@ -10,14 +15,6 @@
 except KeyError:
     pass
 
-import matplotlib
```

Comment:
Okay, seems that we are all on the same page now. This axis-business is something I can't do all day long.

I just tried the cens feature tests with the master-branch normalization and it passes so I would suggest we close this one and figure the normalization out in a separate PR?
@nils-werner and I can sit together tomorrow and propose some new normalization which then might break the way the axis along which to normalize is specified to enable higher dimensions.

@bmcfee what do you think?

Alternative: Only allow 2d normalization.